### ISSUE TITLE
Smtp don't work

### ISSUE THREAD

**Issue by arckosfr** [thp/urlwatch#56]
```python
watched 6 URLs in 0 seconds
Traceback (most recent call last):
  File "/usr/local/bin/urlwatch", line 375, in <module>
    main(parser.parse_args())
  File "/usr/local/bin/urlwatch", line 342, in main
    report.finish()
  File "/usr/local/lib/python3.4/dist-packages/urlwatch/handler.py", line 128, in finish
    ReporterBase.submit_all(self, self.job_states, duration)
  File "/usr/local/lib/python3.4/dist-packages/urlwatch/reporters.py", line 81, in submit_all
    cls(report, cfg, job_states, duration).submit()
  File "/usr/local/lib/python3.4/dist-packages/urlwatch/reporters.py", line 340, in submit
    mailer.send(msg)
  File "/usr/local/lib/python3.4/dist-packages/urlwatch/mailer.py", line 57, in send
    s.starttls()
  File "/usr/lib/python3.4/smtplib.py", line 689, in starttls
    server_hostname=server_hostname)
  File "/usr/lib/python3.4/ssl.py", line 364, in wrap_socket
    _context=self)
  File "/usr/lib/python3.4/ssl.py", line 577, in __init__
    self.do_handshake()
  File "/usr/lib/python3.4/ssl.py", line 804, in do_handshake
    self._sslobj.do_handshake()
ssl.SSLError: [SSL: TLSV1_UNRECOGNIZED_NAME] tlsv1 unrecognized name (_ssl.c:600)
```

i have tried with gmail server and own mailserver, with port 25/465/587, and with/without starttls
on multiple server

**Comment by brbsix** [comment]
Have you updated to the latest version of urlwatch? AFAIK that issue has been resolved. See #50.

**Comment by arckosfr** [comment]
i used version on pipy because the github version on docker build weird.

1 mars 2016 17:15 "Six"  a écrit:
	Have you updated to the latest version of urlwatch? AFAIK that issue has been resolved. See #50 (https://github.com/thp/urlwatch/issues/50).

**Comment by brbsix** [comment]
Well if you can't install the latest sources from GitHub (presumably via `pip install --user --upgrade git+https://github.com/thp/urlwatch` or something along those lines) then just patch *urlwatch/mailer.py* of your existing install.

```
@@ -49,8 +49,7 @@ def __init__(self, smtp_server, smtp_port, tls, auth):
          self.auth = auth
  
      def send(self, msg):
 -        s = smtplib.SMTP()
 -        s.connect(self.smtp_server, self.smtp_port)
 +        s = smtplib.SMTP(self.smtp_server, self.smtp_port)
          s.ehlo()
  
          if self.tls:
```

**Comment by thp** [comment]
Closing as fixed / duplicate of #50
